src/core/Notification/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { AlertProps } from "@material-ui/lab";
+import React from "react";
+import { ReactComponent as IconAlert } from "../../common/svgs/IconAlert.svg";
+import { ReactComponent as IconClose } from "../../common/svgs/IconClose.svg";
+import { ReactComponent as IconSuccess } from "../../common/svgs/IconSuccess.svg";
+import { StyledButton, StyledNotification } from "./style";
+
+export interface NotificationProps {
+  buttonOnClick?: (event: React.SyntheticEvent) => void;
+  buttonText?: string;
+  intent: "info" | "error" | "success" | "warning";
+}
+
+export type ExposedNotificationProps = AlertProps & NotificationProps;
+
+const Notification = ({
+  children,
+  intent,
+  onClose,
+  buttonOnClick,
+  buttonText,
+  ...rest
+}: ExposedNotificationProps): JSX.Element => {
+  return (
+    <StyledNotification
+      {...rest}
```

Comment:
Ah I think `{...rest}` needs to go last in order to allow the callsite to overwrite props!

Example of Maya's [recent fix](https://github.com/chanzuckerberg/sci-components/pull/67/files)

E.g.,

```tsx
<StyledNotification
  action={...}
  ...
  {...rest}
>

```